inappmessaging/src/test/java/com/rakuten/tech/mobile/inappmessaging/runtime/service/DisplayMessageJobIntentServiceSpec.kt

```diff
@@ -217,6 +215,52 @@ class DisplayMessageJobIntentServiceSpec : BaseTest() {
         Mockito.verify(activity, never()).findViewById<View?>(ArgumentMatchers.anyInt())
     }
 
+    @Test
+    fun `should display message with image`() {
+
+        val imageUrl =
+        "https://iamprodjpefiles.blob.core.windows.net/campaign-images/e16c43ba-c4e2-491c-95d0-9444a6d63c91"
```

Comment:
Avoid using this URL for test code. Use public image URL.

Reply:
Updated the URL to use public one.